pagopa/io-web-profile
# [IOPID-521] : L1, L2, L3 redirect to correct page

## Short description

Upon successful login from the IdP, we intercept the token to verify the authentication level and subsequently redirect users to the appropriate page based on the flow and authentication level. This process is managed in the "validateSession" page.

## List of changes proposed in this pull request

Added L3 Login Page (previously missing) [IOPID-555]
Added Restore Profile Page from L3 access page
Added redirect logic in validateSession page and utils [IOPID-521]

## How to test

Attempt to log in through **/access** (L2), **/logoutInit** (L1), and **/accessL3** (L3). After login, you should be automatically redirected to different pages based on your authentication level. (Note: For accessing /accessL3, prior login with L2 credentials is required as per the requirements).

To log in, use hub-spid-login


[IOPID-555]: https://pagopa.atlassian.net/browse/IOPID-555?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IOPID-555]: https://pagopa.atlassian.net/browse/IOPID-555?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IOPID-521]: https://pagopa.atlassian.net/browse/IOPID-521?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
